ceph-pr-combined/config/definitions/ceph-pr-combined.yml

```diff
@@ -0,0 +1,135 @@
+- job:
+    name: ceph-pr-combined
+    description: 'make check and ceph API tests combined'
+    project-type: freestyle
```

Comment:
Not sure if this would imply much change to everything else, but we might keep the Jenkins Job Builder YAML syntax while introducing [`project-type: pipeline`](https://docs.openstack.org/infra/jenkins-job-builder/project_pipeline.html)...

Reply:
Yeah, I'm sure in the future we can streamline this job by making pipeline.  The two troubles I had with that were

1. Ensuring all steps run on the same machine
2. Having two separate "checks" in GitHub and having them get updated accordingly

Reply:
Thanks!